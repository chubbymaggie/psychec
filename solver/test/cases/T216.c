struct TUPLE split(NODE_PTR a) {
    struct TUPLE x = split(a->next);
    return x;
}

void merge_sort(NODE_PTR c)
{
    struct TUPLE s = split(c);
}
