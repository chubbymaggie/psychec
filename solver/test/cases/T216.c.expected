/* Forward declarations */
typedef  struct TYPE_4__   TYPE_1__ ;

/* Type definitions */
struct TUPLE {int dummy; } ;
struct TYPE_4__ {struct TYPE_4__* next; } ;
typedef  TYPE_1__* NODE_PTR ;

/* Variables and functions */

